### DISCUSSION TITLE
Can Gzip unarchive return Member?

### DISCUSSION THREAD

**Discussion by HouHangLei** [tsolomko/SWCompression#35] [Q&A] [answered]
After unzipping I need to get the file name (mainly the file suffix) so it can be stored in the sandbox directory.

**Comment by tsolomko** [comment, marked as answer]
Yes, you can use [`GzipArchive.multiUnarchive(archive:)`](https://www.tsolomko.me/SWCompression/Classes/GzipArchive.html#/s:13SWCompression11GzipArchiveC14multiUnarchive7archiveSayAC6MemberVG10Foundation4DataV_tKFZ) instead. It works even on a single GZip archive, in which case it returns an array with only a single `Member`.
